packages/vest/src/utilities/tests/promisify.test.js

```diff
@@ -0,0 +1,82 @@
+import faker from 'faker';
+import vest from '../..';
+import { dummyTest } from '../../../testUtils/testDummy';
+import { promisify } from '../promisify';
+
+describe('Utility: promisify', () => {
+  let validatorFn;
+  let validateAsync;
+
+  beforeEach(() => {
+    validatorFn = jest.fn(() => vest.validate('test-promisify', jest.fn()));
+    validateAsync = promisify(validatorFn);
+  });
+
+  describe('Test arguments', () => {
+    it('Should throw an error', () => {
+      const invalidValidateAsync = promisify('invalid');
+      expect(() => invalidValidateAsync()).toThrow();
+    });
+  });
+
+  describe('Return value', () => {
+    it('should be a function', () => {
+      expect(typeof promisify(jest.fn())).toBe('function');
+    });
+
+    it('should be a promise', () =>
```

Comment:
i think u wanted to pass `done` here

Reply:
https://github.com/jest-community/eslint-plugin-jest/blob/HEAD/docs/rules/no-test-callback.md